extras/lbryinc/client.go

```diff
@@ -13,11 +13,18 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-// Client stores data about internal-apis call it is about to make.
+// Caller interface defines currently supported internal-apis methods.
+type Caller interface {
```

Comment:
So when naming interfaces, those that end with an `er` should only have one method and it should be that remaining string. This is the convention for single method interfaces. So if you are naming an interface `Caller` it should only specify one method and it should be called `Call`.

Reply:
I did not know this

Reply:
https://golang.org/doc/effective_go.html#interface-names for reference